ddsketch/ddsketch.go

```diff
@@ -181,6 +181,19 @@ func (s *DDSketch) GetMinValue() (float64, error) {
 	}
 }
 
+// GetSum returns the sum of all values in the sketch.
```

Comment:
Can you add comments as to how accurate that is (similar to what we have in the Java implementation)?

Reply:
Added a comment. I haven't found the comment in Java though, so let me know if that works.

Reply:
It's [there](https://github.com/DataDog/sketches-java/blob/master/src/main/java/com/datadoghq/sketch/ddsketch/DDSketch.java#L266:L268), but I think that works too.